README.md

```diff
@@ -16,28 +16,55 @@ To guide development we [have documented the user needs](docs/NEEDS.md) and [ass
 
 ## Usage
 
+To run a GOV.UK application, `govuk-docker build` it first (you only need to do this once per application):
+
 ```sh
-# Run a Rake task on Whitehall
-whitehall$ govuk-docker run bundle exec rake -T
+# This assumes you're in the ~/govuk/govuk-docker directory.
+$ govuk-docker build --service collections-publisher
+```
 
+Then run it with `govuk-docker startup`:
+
+```sh
 # Start content-tagger including dependencies. Visit it at content-tagger.dev.gov.uk
-content-tagger$ govuk-docker startup
+$ govuk-docker startup --service collections-publisher
+```
+
+Alternatively, you can run these `govuk-docker` commands directly from the application directory. If no `service` parameter is provided, govuk-docker matches the name of the current directory with the [corresponding docker-compose.yml in govuk-docker/services](https://github.com/alphagov/govuk-docker/blob/master/services/content-tagger/docker-compose.yml):
+
+```sh
+$ cd ~/govuk/content-tagger && govuk-docker startup
+```
+
+`govuk-docker startup` runs the application on the `app` [stack](#stacks) by default, but you can override the stack by passing an unnamed parameter which will be taken as the stack name, with an `app-` prefix. Example:
+
+```sh
+# Start content-publisher plus the "app-e2e" (end-to-end) stack
+$ govuk-docker startup --service content-publisher e2e
+```
+
+Another useful govuk-docker command is `run`:
 
-# Start content-publisher plus an "end-to-end" stack
-content-publisher$ govuk-docker startup e2e
+```sh
+# Run a Rake task on Whitehall
+$ cd ~/govuk/whitehall && govuk-docker run bundle exec rake -T
 ```
 
+For a full list of govuk-docker commands, run `govuk-docker help`.
+
 ## Installation
 
 ### Prerequisites
 
-First make sure the following are installed on your system:
+govuk-docker has the following dependencies:
 
-- [dnsmasq](http://www.thekelleys.org.uk/dnsmasq/doc.html) to make *$app-name.dev.gov.uk* work. You can install this using `brew install dnsmasq`.
-- [docker](https://hub.docker.com/) and [docker-compose](https://docs.docker.com/compose/install/)
-- [git](https://git-scm.com) if you're setting everything up from scratch
+- [brew](https://brew.sh/)
```

Comment:
There are some developers on GOV.UK who don't use a Mac, in which case installing Brew won't be possible for them, but `govuk-docker` itself will still work. For them, they'll have to manually install dnsmasq themselves. It might be good to add something about that here.